docs/tutorials/data_management.md

```diff
@@ -83,13 +78,13 @@ fn main() {
 
 # 4. Data slicing
 
-You might have noticed that the blob we generated is a hefty 13MB. This is no surprise, as we included `--all-keys` `--all-locales`. However, our binary only uses date formatting data in Japanese. There's room for optimization:
+You might have noticed that the blob we generated is a hefty 13MB. This is no surprise, as by default all keys and for all locales are included. However, our binary only uses date formatting data in Japanese. There's room for optimization:
 
 ```console
-$ icu4x-datagen --overwrite --cldr-tag latest --icuexport-tag latest --out my_data_blob.postcard --format blob --keys-for-bin target/debug/myapp --locales ja
+$ icu4x-datagen --format blob --out my_data_blob.postcard --overwrite --keys-for-bin target/debug/myapp --locales ja
```

Comment:
Observation: There are no longer any examples in this tutorial of how to use `--icuexport-tag` and `--cldr-tag`

Reply:
I think those are niche settings and clients should not generate outdated data.